### PR TITLE
Migrate multicast content - OSDOCS 381

### PR DESCRIPTION
**Pull request by jboxman**
- OSDOCS-381

### CODE REVIEW

**Review by pecameron — APPROVED** [review]
/lgtm

**Comment by jboxman** [comment]
@openshift/team-documentation PTAL, thanks!

File: modules/nw-disabling-multicast.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+.Prerequisites
+
+* The `oc` CLI program installed.
+
+.Procedure
+
+. Disable multicast by running the following command:
++
+[source, bash]
```
Comment (bergerhoffer) [inline modules/nw-disabling-multicast.adoc]: We don't use [source,bash] anymore, just leave this off.

File: modules/nw-enabling-multicast.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.Prerequisites
+
+* The `oc` CLI program installed.
+
+.Procedure
+
+. Run the following command:
++
+[source, bash]
```
Comment (bergerhoffer) [inline modules/nw-enabling-multicast.adoc]: Same about not using source,bash.

File: modules/nw-about-multicast.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+// * networking/using-multicast.adoc
+
+[id="about-multicast-{context}"]
+= About multicast
+
+With IP multicast, data is broadcast to many IP addresses simultaneously.
+
+[IMPORTANT]
+====
+At this time, multicast is best used for low bandwidth coordination or service
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: low bandwidth should probably be hyphenated (high-bandwidth is later on)

File: modules/nw-about-multicast.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+discovery and not a high-bandwidth solution.
+====
+
+Multicast traffic between {product-title} pods is disabled by default. If you
+are using the *OpenShiftSDN* network plug-in, you can enable multicast on a
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: As far as I know, we only use asterisks for items in the UI, not just to highlight something. I don't think you should use any markup around this

File: modules/nw-about-multicast.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+per-project basis.
+
+When using the *OpenShiftSDN* network plug-in:
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: Same here about not using asterisks

File: modules/nw-about-multicast.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+. Multicast packets sent by a pod will be delivered to all other pods in the
+project, regardless of `NetworkPolicy` objects. (Pods may be able to communicate
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: I would remove the parentheses around this sentence.

File: modules/nw-about-multicast.adoc [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+over multicast even when they can't communicate over unicast.)
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: can't -> cannot

File: modules/nw-about-multicast.adoc [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+. Multicast packets sent by a pod in one project will never be delivered to pods
+in any other project, even if there are `NetworkPolicy` objects allowing
+communication between the to projects.
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: Should "to" be "two"? Or just remove the "to" to say "between the projects"?

File: modules/nw-disabling-multicast.adoc [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+You can disable multicast between pods for your project.
+
+.Prerequisites
+
+* The `oc` CLI program installed.
+
+.Procedure
+
+. Disable multicast by running the following command:
```
Comment (bergerhoffer) [inline modules/nw-disabling-multicast.adoc]: If it's only one step, it should be a regular bullet.

File: modules/nw-enabling-multicast.adoc [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+You can enable multicast between pods for your project.
+
+.Prerequisites
+
+* The `oc` CLI program installed.
+
+.Procedure
+
+. Run the following command:
```
Comment (bergerhoffer) [inline modules/nw-enabling-multicast.adoc]: Same about using a regular bullet if it's just one step

File: modules/nw-about-multicast.adoc [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+discovery and not a high-bandwidth solution.
+====
+
+Multicast traffic between {product-title} pods is disabled by default. If you
+are using the *OpenShiftSDN* network plug-in, you can enable multicast on a
+per-project basis.
+
+When using the *OpenShiftSDN* network plug-in:
+
+. Multicast packets sent by a pod will be delivered to all other pods in the
```
Comment (bergerhoffer) [inline modules/nw-about-multicast.adoc]: I think these should be regular bullets, because it's not a set of steps or something that happens in a particular order.

File: modules/nw-enabling-multicast.adoc [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+// * networking/using-multicast.adoc
+
+[id="enabling-multicast-{context}"]
+= Enabling multicast between pods
+
+You can enable multicast between pods for your project.
+
+.Prerequisites
+
+* The `oc` CLI program installed.
```
Comment (bergerhoffer) [inline modules/nw-enabling-multicast.adoc]: I would reword this to say "Install the `oc` CLI program." Though instead of "program", I would either say "tool" or just leave it off and just say "`oc` CLI".

File: modules/nw-disabling-multicast.adoc [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+// * networking/using-multicast.adoc
+
+[id="disabling-multicast-{context}"]
+= Disabling multicast between pods
+
+You can disable multicast between pods for your project.
+
+.Prerequisites
+
+* The `oc` CLI program installed.
```
Comment (bergerhoffer) [inline modules/nw-disabling-multicast.adoc]: Same as I mentioned elsewhere about rewording this.

**Review by bergerhoffer — COMMENTED** [review]
Just a few things. Also, this PR needs rebased to resolve the conflicts.

File: modules/nw-about-multicast.adoc (hunk 8 above)
Comment (jboxman) [reply modules/nw-about-multicast.adoc]: The latter makes the most sense, thanks!

**Comment by jboxman** [comment]
@pecameron @danwinship @squeed do you know what role a user needs to run `oc annotate netnamespace`? Can project admins do this, or only cluster admins?

File: modules/nw-enabling-multicast.adoc (hunk 10 above)
Comment (jboxman) [reply modules/nw-enabling-multicast.adoc]: World's simplest procedure!

File: modules/nw-enabling-multicast.adoc (hunk 12 above)
Comment (jboxman) [reply modules/nw-enabling-multicast.adoc]: I'm curious about the best way to handle this, so I added an item to the agenda.

**Comment by jboxman** [comment]
@bergerhoffer thanks for looking at this!

**Comment by danwinship** [comment]
Only cluster admins can annotate namespaces

**Comment by jboxman** [comment]
@vikram-redhat, I think that this is ready to be merged. Do I need to be aware of anything in particular, once I've rebased and resolved the merge conflict with `_topic_map.yml`?

**Comment by vikram-redhat** [comment]
@jboxman the only thing to check would be to see if any of these stories apply to the dedicated and online distros and to exclude them. This applies to not just this story, but all stories that you are working on. 

cc: @bmcelvee

**Comment by jboxman** [comment]
@vikram-redhat, do you mean to exclude the content entirely if it _does not_ apply to _dedicated_ and/or _online_?

**Comment by vikram-redhat** [comment]
@jboxman yeah. There are some stories where only a part of the story may apply to those distros. I don't know if we have the bandwidth to do that at the moment, but if possible that will good to do as well.

**Comment by jboxman** [comment]
@vikram-redhat, none of these stories apply to OpenShift Dedicated in the current docs.

**Comment by vikram-redhat** [comment]
@jboxman gotcha. Exclude them all.

**Comment by jboxman** [comment]
@vikram-redhat, I checked further. It should be possible to set network policy objects. I'll exclude the rest.
